packages/http-server/src/server.ts

```diff
@@ -19,24 +55,7 @@ export const createServer = <LoaderInput>(
     logger: (components && components.logger) || createLogger('HTTP SERVER'),
     disableRequestLogging: true,
     modifyCoreObjects: false,
-  })
```

Comment:
If `application/json` was first in `content-type` from request (like `application/json,application/xml`), it was ignored if there was a manually added serializer for `application/xml`.
Since the serializer for `JSON`, was the default one, I decided to follow that path and not add a serializer for JSON to `serializers`. Instead I moved out serializers out of `fastify`.